api/tests/unit/domain/usecases/manually-resolve-certification-issue-report_test.js

```
@@ -59,5 +62,32 @@ describe('Unit | UseCase | manually-resolve-certification-issue-report', functio
         expect(certificationIssueReportRepository.save).to.have.been.calledWith(expectedCertificationIssueReport);
       });
     });
+
+    describe('when certification issue report is resolved automatically', function () {
+      it('should not save certification issue report', async function () {
```

Suggestion: should throw an error